### ISSUE TITLE
Provide icons for all default resource formats

### ISSUE THREAD

**Issue by sheldonrampton** [GetDKAN/dkan#982]
CIVIC-3211

## Description

When viewing lists of datasets, the little icons reflecting data resource format are correctly displayed, e.g., rest for REST data and csv for CSV data. However, when viewing an individual dataset, the icons for REST data are not correct, e.g. a generic "data" icon displays.

This issue has been documented for DKAN 7.x-1.11.
## Steps to Reproduce

Go to a test dataset named "Solid Waste Facilities-Municipal" that I have set up on a fresh install of DKAN 7.x-1.11.

http://dev-dkan-111.getnucivic.com/dataset/solid-waste-facilities-municipal

The dataset contains three resources:
- "Web services" has a data format of "rest."
- "Data Table" has a data format of "csv."
- "Download Data" has a data format of "data."

However, the icons which you see on the dataset page are the same for both "Web Services" and "Download Data":

![solid_waste_facilities-municipal___test](https://cloud.githubusercontent.com/assets/2838758/13686560/f112dcca-e6db-11e5-949d-73d8fcf6cde1.png)

In other words, the icons for Download Data and Data Table are correct, but having the Web Services item also use the data icon is not correct. It should have a different icon specific to REST data.

You can view the same dataset within a search listing page by searching for the word "solid." If you do so, you will note that each listed data resource has a different name, specific to its format.
## Acceptance Criteria
- Data formats included within the standard set DKAN taxonomy set of data formats should each have their own unique icons.
## Ticket Reference

NCS-12

**Comment by janette** [comment]
the icons rely on the user filling in the 'format' field correctly

![screenshot_3_11_16__10_32_am](https://cloud.githubusercontent.com/assets/314172/13708507/9434af62-e774-11e5-9f5a-59ea8f5ed013.png)

**Comment by acouch** [comment]
I don't think I'd qualify this as a bug but a feature request. "Data" is the default. We don't have a mechanism yet for discovery of some file types.

This issue and the linked PRs add a better mechanism for file type discovery that includes many more types and further limits the use of the generic "data": https://github.com/NuCivic/dkan/issues/739

**Comment by dafeder** [comment]
Also, for better or worse we use an actual icon image in the dataset display, where we use text in the dataset list. We might want to try to do something in the case of default/data dispaly that would display the actual format name in a way that mimics the icon format.

**Comment by acouch** [comment]
Closing this ticket as it has not been updated recently and is not on our near-term roadmap. Please re-open if you would like to discuss further or are actively working on it.
